opentelemetry-api/src/trace/noop.rs

```diff
@@ -31,9 +31,9 @@ impl trace::TracerProvider for NoopTracerProvider {
     /// Returns a new `NoopTracer` instance.
     fn versioned_tracer(
         &self,
-        _name: impl Into<Cow<'static, str>>,
-        _version: Option<&'static str>,
-        _schema_url: Option<&'static str>,
+        _name: Cow<'static, str>,
+        _version: Option<Cow<'static, str>>,
```

Comment:
Curious as to why we are removing the `&` for `version` and `schema_url`? Are we allowing accepting of both `String` and `&str` for these fields?

Reply:
Yes previously it only allowed for static strings, but here it allows for `Cow<'static, str>`, which means it can be an owned string if the version is not static (eg might come from user input, io, etc).